AspNetCoreWeb/Infrastructure/SearchOptionsProcessor.cs

```diff
@@ -51,7 +51,7 @@ private static IEnumerable<SearchTerm> GetValidTerms(IEnumerable<DTColumn> colum
                 yield break;
             }
 
-            var declaredTerms = GetTermsFromModel(typeof(TModel));
+            var declaredTerms = GetTermsFromModel(typeof(TModel)).ToList();
```

Comment:
why a `.ToList()`?

Reply:
Since the below line of code at line 58 checks through the Enumerable each time I thought it would be best if it was using a List instead.

`var declaredTerm = declaredTerms.SingleOrDefault(x => x.Name.Equals(term.Name, StringComparison.OrdinalIgnoreCase));`